hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeElement.java

```diff
@@ -430,13 +430,15 @@ public static final ElementKeys classifyKeys(Object... keyValues) {
                 throw Element.Exceptions
                       .providedKeyValuesMustHaveALegalKeyOnEvenIndices();
             }
-            if (val == null) {
-                throw Property.Exceptions.propertyDoesNotExist();
-            }
 
             if (key.equals(T.id)) {
                 elemKeys.id = val;
             } else if (key.equals(T.label)) {
+
+                if(val==null){
+                    throw Element.Exceptions.labelCanNotBeNull();
```

Comment:
prefer to keep the `throw Property.Exceptions.propertyDoesNotExist();` and throw labelCanNotBeNull() if  `key.equals(T.label)`.